drivers/openstack/openstack.go

```diff
@@ -736,7 +736,37 @@ func (d *Driver) waitForSSHServer() error {
 		"MachineId": d.MachineId,
 		"IP":        ip,
 	}).Debug("Waiting for the SSH server to be started...")
-	return ssh.WaitForTCP(fmt.Sprintf("%s:%d", ip, d.SSHPort))
+
+	err = ssh.WaitForTCP(fmt.Sprintf("%s:%d", ip, d.SSHPort))
```

Comment:
@ggiamarchi how about you just create the new retry wait.  then, as @ctrlrsf said, we can update the others as needed.